src/libs/EmojiUtils.js

```diff
@@ -201,7 +202,12 @@ function replaceEmojis(text, isSmallScreenWidth = false) {
     for (let i = 0; i < emojiData.length; i++) {
         const checkEmoji = emojisTrie.search(emojiData[i].slice(1, -1));
         if (checkEmoji && checkEmoji.metaData.code) {
-            let emojiReplacement = checkEmoji.metaData.code;
+            let emojiReplacement;
+            if (preferredSkinTone >= 0) {
+                emojiReplacement = checkEmoji.metaData.types[preferredSkinTone];
+            } else {
+                emojiReplacement = checkEmoji.metaData.code;
+            }
```

Comment:
```suggestion
            let emojiReplacement = lodashGet(checkEmoji.metaData.types, preferredSkinTone, checkEmoji.metaData.code);
```

Some emojis does not support types such as `:smile:` and will crash the app.

**IMPORTANT**: Don't forget to import `lodashGet`